conftest.py

```diff
@@ -108,6 +108,13 @@ def run_and_wait(
         assert rv == 0 or rv["StatusCode"] == 0
         return logs
 
+    def get_host_port(self, container_port: str) -> str:
+        assert isinstance(self.container, Container)
```

Comment:
Some kind of docstring is always helpful!

```suggestion
        """Returns the host port associated with the tracked container's port."""
        assert isinstance(self.container, Container)
```